redis-exercises/shoppingcart/README.md

```diff
@@ -102,7 +102,7 @@ Now, it's time to get down to business. We decided at the beginning of the exerc
 ```diff
 > function.js
 ---
- exports.handler = async (body, context) => {
+ exports.addItem = async (body, context) => {
```

Comment:
The diff is wrong here

Reply:
I think you're getting confused between actual diffs and README diffs

Reply:
Just to make sure we're on the same page, the diff is wrong since you start out with:
```
exports.handler = ...
```
and end up with:
```
exports addItem = ...
```